andela/izanagi-rc
# Feature/142658237/cancel order option

#### What does PR do ?

Integrates the cancel order button for both a user and a vendor

#### Description of Task to be completed?

As a user, I would like the option to choose "cancel order" and have my order cancelled. As an Admin/Vendor I should be able to view every order cancelled, and the final authorization to cancel the order and refunded depending on status must lie with me.

#### How should this be manually tested?

As a user when you open up the app, make an order, then try cancelling the order, As an Admin I should be able to cancel orders.

#### Any background context you want to provide?

N/A

#### Questions:

N/A

#### Screenshots

N/A
